app/javascript/components/pages/ApartmentShow.js

```diff
@@ -4,10 +4,10 @@ import { Button } from 'react-bootstrap'
 import { useParams } from 'react-router-dom'
 
 const ApartmentShow = props => {
-  const { apartments } = props
+  const { apartments, logged_in, current_user } = props
   let { id } = useParams()
   let apartment = apartments.find(apartment => apartment.id == id)
-
+  let usersApartment = apartments.find(apartment => apartment.id == id && apartment.user_id === current_user.id) 
```

Comment:
I have been informed that the proper thing to do with these params is to use strict equality. 
```javascript
let apartment = apartments.find(apartment => apartment.id === +id)
  let usersApartment = apartments.find(apartment => apartment.id === +id && apartment.user_id === current_user.id)